uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/document-list.js

```diff
@@ -19,37 +19,28 @@ define([
 
       var defaultDocumentStatus = ['1', '2']; // 1: 'awaiting upload' | 2: 'awaiting approval
```

Comment:
Are you sure `defaultDocumentStatus` is a good name for this variable. Since it is not used for showing default statuses I think it should be renamed.